build-newlib.sh

```diff
@@ -20,8 +20,14 @@
 build_newlib () {
   pushToCleanDir $BUILD_DIR/newlib
 
+  ENABLE_FP="--disable-newlib-hw-fp"
```

Comment:
I think this could be better named. We could have a case where ENABLE_FP="--disable-..." which is counter intuitive.

Maybe simply NEWLIB_HW_FP?

Reply:
yes, good idea. done.